frontend/src/static/js/v_summary.js

```diff
@@ -271,7 +271,8 @@ window.vSummary = {
       if (this.filterGroupWeek) {
         sinceDate = dateRounding(sinceDate, 1);
       }
-      let diff = getIntervalDay(userFirst.sinceDate, sinceDate);
+      let diff = userFirst ?
```

Comment:
> const userFirst = user.dailyCommits[0];

In the event that user.dailyCommits is an empty array, wouldn't it cause an invalid index error?

Reply:
javascript behavior is to return `undefined`

Reply:
oh, actually on second thoughts, i might rewrite the fix to play around this idea of an invalid index.